core/src/main/java/com/digitalpebble/stormcrawler/persistence/AdaptiveScheduler.java

```diff
@@ -145,9 +145,7 @@
 
     /**
      * Name of key to hold previous signature: a copy, not overwritten by {@link
-     * MD5SignatureParseFilter}, is added by {@link
```

Comment:
why removing part of the Javadoc?

Reply:
Because `com.digitalpebble.stormcrawler.parse.filter.SignatureCopyParseFilter` does not exist. The rest of the comment was deleted by a mistake. I'll revert the rest.

Reply:
ah, makes sense. Thanks